packages/v4/patternfly-docs/content/design-guidelines/components/code-editor/code-editor.md

```diff
@@ -58,4 +58,7 @@ Use the read-only state of the code editor when you only want to display large b
 
 <img src="./img/code-editor-read-only.png" alt="Example of a read only code-editor" width="800"/>
 
-The border of the code editor will turn gray to indicate that it is read-only. The user can still highlight text in the editor, but won’t be able to edit. Adding actions is optional and should be determined by use case.
\ No newline at end of file
+The border of the code editor will turn gray to indicate that it is read-only. The user can still highlight text in the editor, but won’t be able to edit. Adding actions is optional and should be determined by use case.
+
+## Accessibility 
+For information regarding accessibility, visit the [code editor accessibility](/components/code-editor/accessibility) tab. 
```

Comment:
The only comment is that the word "tab" should be included in the link! Otherwise looks great!